errata_tool/variant.py

```diff
@@ -0,0 +1,24 @@
+from errata_tool import ErrataConnector
+
+
+class Variant(ErrataConnector):
+    def __init__(self, name, data=None):
+        self.name = name
+        self.data = data
+        self.url = self._url + '/variants/%s' % self.name
+
+    def refresh(self):
+        url = self.url + '.json'
```

Comment:
I'm thinking we should use the newer REST API instead, because the ET dev team does not really support this older ".json" API very much anymore. (I'm not sure that's documented, but it's effectively true)

For example https://errata.devel.redhat.com/api/v1/variants/8Base-RHCEPH-5.0-Tools

Reply:
The JSON output appears to differ between the old and new API endpoints:

https://errata.devel.redhat.com/variants/8Base-RHCEPH-5.0-Tools.json
https://errata.devel.redhat.com/api/v1/variants/8Base-RHCEPH-5.0-Tools

Normally this is a non-issue, but I'm not sure if there's a v1 API equivalent to the '.json' API for retrieving https://errata.devel.redhat.com/product_versions/RHCEPH-5.0-RHEL-8/variants.json

I believe it's important to ensure properties don't change whenever __getattr__ or refresh() are invoked. What do you think?